boxoffice/templates/ticket_reassignment_mail.html.jinja2

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>{{ org.title }}</title>
+  <style type="text/css">
+    * {
+      margin: 0;
+      padding: 0;
+    }
+
+    body {
+      font-family: "Source Sans Pro",Arial,sans-serif;
+      font-size: 16px;
+      line-height: 1.2;
+      color: #333;
+      background-color: #fff
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    p {
+      font-size: 15px;
+      margin: 5px 0;
+    }
+
+    .text-center {
+      text-align: center;
+    }
+
+    .wrapper {
+      background: #EBD3A3;
+      text-decoration: none;
+      padding: 10px;
+      max-width: 500px;
+      margin: 20px 0;
+      border-radius: 5px;
+    }
+
+    .ticket-table {
+      width: 100%;
+      border-collapse: collapse;
+    }
+
+    .ticket-table td {
+      border: 2px solid #cf6f01;
+    }
+
+    .ticket-title-wrapper {
+      max-width: 180px;
+      padding: 10px 10px 0;
+    }
+
+    .ticket-title {
+      font-size: 18px;
+      font-weight: bold;
+      margin-bottom: 20px;
+    }
+
+    .receipt-no {
+      color: #941f2d;
+      font-size: 13px;
+      font-style: italic;
+      margin: 5px 0 0;
+    }
+
+    .details-wrapper {
+      padding: 10px;
+    }
+  </style>
+</head>
+
+<body>
+  <div>
+
+    <p>Hello {{ old_assignee.fullname }},</p>
+
+    <div class="wrapper">
+      <table class="ticket-table">
+        <tr>
+          <td class="ticket-title-wrapper">
+            <p class="text-center ticket-title">{{ line_item.item.title }}</p>
+            <p class="receipt-no">Receipt No: {{ order.invoice_no }}</p>
+          </td>
+
+          <td class="details-wrapper">
+          <p>
+            {% if current_auth.user %}
+              {% trans new_assignee=new_assignee.fullname, current_user=current_auth.user.fullname %}
+                Your ticket has been transfered to {{ new_assignee }}, by {{ current_user }}.
+              {% endtrans %}
+            {% else %}
+              {% trans new_assignee=new_assignee.fullname %}
+                Your ticket has been transfered to {{ new_assignee }}.
+              {% endtrans %}
+            {% endif %}
+          </p>
+
+          <p>
+            {% if old_assignee.email != order.buyer_email %}
+              {% trans buyer_email=order.buyer_email, buyer_fullname=order.buyer_fullname %}
+                If you believe this was because of an error, please reach out to <a href="mailto:{{ buyer_email }}">{{ buyer_fullname }}</a>.
+              {% endtrans %}
+            {% else %}
+              {% trans support_email = org.details.get('support_email', '') %}
+                If you believe this was because of an error, please reach out to <a href="mailto:{{ support_email }}">{{ support_email }}</a>.
```

Comment:
If we are sharing the buyer's email with the assignee, then we should warn in the buyer in the assignment form. Or remove the email here.

Reply:
How do we warn the buyer? Because i think it happens often that someone buys all the tickets and then assigns them one by one. and the asignees get mail. but then if their ticket gets reassigned to someone else without their own consent, what course of action do they have?

Reply:
Whoever has access to the assignment form should have a notice displayed: when you assign this ticket, the buyer's email will be shared. Or we should remove the email here. The name alone should be sufficient to help the assignee recognise the assigner.

Reply:
Alright. I'll remove the email and keep the name.